src/main/java/webDriver/ChromeRunner.java

```diff
@@ -0,0 +1,21 @@
+package webDriver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class ChromeRunner {
+    public static void main(String[] args) {
+        initChromeDriver();
+    }
+
+    public static WebDriver initChromeDriver() {
+        String exePath = "C:\\Users\\Simba\\Downloads\\chromedriver_last\\chromedriver.exe";
+        System.setProperty("webdriver.chrome.driver", exePath);
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
```

Comment:
try to avoid using implicit wait timeout. This is driver's timeout and will affect all tests in scope of this driver instance. Also it affects explicit timeout and can cause unexpected wait failures.
Example:
You set 5 seconds implicit.
then create explicit like
```new WebDriverWait(driver, 10L).pollingInterval(2L, TimeUnit.SECONDS).ignore(NoSuchElementException.class).until(() -> driver.findElement("some xpath").isPresent())```
 
It is going to work as follow: Every two seconds lamda expression in until method will be executed. It will ignore NoSuchElementException so we don't care it throws. Since you set implicit wait 5 seconds `driver.findElement("some xpath")` will wait 5 seconds! if there is no element. Then it will repeat `until` but only once! because you set implicit 5 seconds. But you expect it runs 5 times! -- `new WebDriverWait(driver, 10L).pollingInterval(2L, TimeUnit.SECONDS)`

P.S. Ignore mistakes in code, I didn't check it :). So it may not work as is.